test/functional/page_objects/dashboard_page_controls.ts

```diff
@@ -382,25 +384,27 @@ export class DashboardPageControls extends FtrService {
 
   public async optionsListPopoverGetAvailableOptionsCount() {
     this.log.debug(`getting available options count from options list`);
+    await this.optionsListPopoverWaitForLoading();
     const availableOptions = await this.testSubjects.find(`optionsList-control-available-options`);
     return +(await availableOptions.getAttribute('data-option-count'));
   }
 
   public async optionsListPopoverGetAvailableOptions() {
     this.log.debug(`getting available options from options list`);
+    await this.optionsListPopoverWaitForLoading();
     const availableOptions = await this.testSubjects.find(`optionsList-control-available-options`);
-
-    const suggestionElements = await availableOptions.findAllByClassName(
-      'optionsList__validSuggestion'
-    );
-    const suggestions: { [key: string]: number } = await suggestionElements.reduce(
-      async (promise, option) => {
-        const acc = await promise;
-        const [key, docCount] = (await option.getVisibleText()).split('\n');
-        return { ...acc, [key]: Number(docCount) };
-      },
-      Promise.resolve({} as { [key: string]: number })
-    );
+    const optionsCount = await this.optionsListPopoverGetAvailableOptionsCount();
+
+    const selectableListItems = await availableOptions.findByClassName('euiSelectableList__list');
+    const suggestions: { [key: string]: number } = {};
+    while (Object.keys(suggestions).length < optionsCount) {
+      await selectableListItems._webElement.sendKeys(this.browser.keys.ARROW_DOWN);
+      const currentOption = await selectableListItems.findByCssSelector('[aria-selected="true"]');
+      const [suggestion, docCount] = (await currentOption.getVisibleText()).split('\n');
+      if (suggestion !== 'Exists') {
+        suggestions[suggestion] = Number(docCount);
+      }
+    }
```

Comment:
Since `isVirtualized` is `true` for the suggestions `EuiSelectable`, options are dynamically loaded/unloaded as the user scrolls. So, to get all the available options, I had to get a bit creative by using the arrow keys to navigate through each option until I wrap around back to the beginning of the list. Not ideal, but seems to be the only way to make this happen for a virtualized `EuiSelectable` :+1: